mmv1/third_party/terraform/services/compute/resource_compute_attached_disk_test.go

```diff
@@ -315,3 +315,56 @@ resource "google_compute_attached_disk" "test" {
 }
 `, diskPrefix, count, instanceName)
 }
+
+func TestAccComputeAttachedDisk_diskInterface(t *testing.T) {
+	t.Parallel()
+
+	diskName := fmt.Sprintf("tf-test-%d", acctest.RandInt(t))
+	instanceName := fmt.Sprintf("tf-test-%d", acctest.RandInt(t))
+	importID := fmt.Sprintf("%s/us-central1-a/%s/%s", envvar.GetTestProjectFromEnv(), instanceName, diskName)
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAttachedDiskResource(diskName, instanceName) + testAccComputeAttachedDisk_interface("testInterface", "SCSI"),
+			},
+			{
+				ResourceName:      "google_compute_attached_disk.testInterface",
+				ImportStateId:     importID,
+				ImportState:       true,
+				ImportStateVerify: false,
+			},
+			{
+				Config: testAttachedDiskResource(diskName, instanceName) + testAccComputeAttachedDisk_noInterface("testNoInterface"),
```

Comment:
Can you add a test step that adds an interface where there was not one specified? And one that changes the interface on a disk?

Reply:
made the field foreNew and ignore read and added tests